x-pack/test/plugin_api_integration/test_suites/task_manager/health_route.ts

```diff
@@ -237,7 +237,6 @@ export default function ({ getService }: FtrProviderContext) {
       expect(typeof workload.overdue).to.eql('number');
 
       expect(typeof workload.non_recurring).to.eql('number');
-      expect(typeof workload.owner_ids).to.eql('number');
```

Comment:
this should still be a valid assertion right? we're not removing it from the health report?

Reply:
The `owner_ids` value got removed from the health report when I removed the aggregation in `x-pack/plugins/task_manager/server/monitoring/workload_statistics.ts`. I figured it was no longer worth it given it always returns 0 as a value. I think that's ok?

Reply:
Ah gotcha 👍